agent/engine/common_integ_testutil.go

```diff
@@ -355,6 +372,10 @@ func VerifyTaskStatus(status apitaskstatus.TaskStatus, taskARN string, testEvent
 // This method queries the TestEvents struct to check a Task Status.
 // This method will block if there are no more stateChangeEvents from the DockerTaskEngine but is expected
 func VerifyContainerStatus(status apicontainerstatus.ContainerStatus, ARNcontName string, testEvents *TestEvents, t *testing.T) {
+	if runtime.GOOS == "windows" && status == apicontainerstatus.ContainerManifestPulled {
+		t.Log("Not expecting image manifest pulls on Windows")
```

Comment:
Let's also add why this won't happen on Windows. That is, because our integration test setup on Windows does not involve image pulls and uses cached images instead.